src/app/body/milestone-details/milestone-details.component.ts

```diff
@@ -100,11 +106,12 @@ export class MilestoneDetailsComponent implements OnInit {
             this.milestoneDataReady = false;
             this.sprintNumber = this.startService.currentSprintNumber;
             this.getMilestoneDetails();
-            console.log("hit in else");
```

Comment:
remove console

Reply:
Sanjay bhaiya said, its need to test in production.